server/controllers/events/testfixtures/test-repos/workspace-parallel-yaml/exp-output-apply-all-production.txt

```diff
@@ -1,3 +1,9 @@
+Ran Apply for 2 projects:
```

Comment:
This expected output is very different. Why?

Reply:
ah, this is the test for parallel output. In this test, just check substring. Thank you for review, I'll revert this.

Reply:
done https://github.com/runatlantis/atlantis/pull/2947/commits/4d6af33adf8e1e6f72db178e1d5eea8009efce72